donor-portal/src/pages/layout/info-box/index.tsx

```diff
@@ -20,11 +20,16 @@ export default function InfoBox({
       <img src={iconUrl} alt="" />
       <h2>{heading}</h2>
       <p>{text}</p>
-      {link !== "#" && (
-        <Link to={link}>
-          <button type="button">{buttonText}</button>
-        </Link>
-      )}
+      {link !== "#" &&
+        (!link.startsWith("https://") ? (
+          <Link to={link}>
+            <button type="button">{buttonText}</button>
+          </Link>
+        ) : (
+          <a href={link} target="_blank" rel="noreferrer">
```

Comment:
Do we need to open this in a new tab?

Reply:
I think it's good to open external links in a new tab.